src/qgitrepository.h

```diff
@@ -681,6 +667,31 @@ namespace LibQGit2
              */
             bool shouldIgnore(const QString &path) const;
 
+            /**
+             * Set the identity to be used for writing reflogs.
+             *
+             * @param name The name to use for the reflog entries
+             * @param email The email to use for the reflog entries
+             *
+             * @throws LibQGit2::Exception
+             */
+            void setIdent(const QString &name, const QString &email);
+
+            /**
+             * The identity used for writing reflogs.
+             */
+            struct Ident {
+                QString name;       ///< The name used for the reflog entries
+                QString email;      ///< The email used for the reflog entries
+            };
+
+            /**
+             * Retrieve the configured identity to use for reflogs.
+             *
+             * @throws LibQGit2::Exception
+             */
+            Ident ident() const;
```

Comment:
I would use the whole word, "identity", in the type name as well as in the function names. C programmers usually seem to suffer from lack of characters, but OO-programmers have plenty of them ;D

Reply:
Yeah, I copied the terminology straight from libgit2. Added some character to this function and type.